### ISSUE TITLE
macOS Illegal instruction: 4 (seen on Travis)

### ISSUE THREAD

**Issue by tlepoint** [open-quantum-safe/liboqs#145]
From [https://travis-ci.org/open-quantum-safe/liboqs/jobs/251993063](https://travis-ci.org/open-quantum-safe/liboqs/jobs/251993063), before the fails due to clang-format, we get the following error:

```
./test_kex
================================================================================
Sample computation for key exchange method LWE Frodo recommended
================================================================================
make: *** [test] Illegal instruction: 4
```

I have no idea where it comes from, I never got it on my mac. Anyone? Maybe we should `valgrind`.

**Comment by tlepoint** [comment]
Idem in [https://travis-ci.org/open-quantum-safe/liboqs/jobs/252002293](https://travis-ci.org/open-quantum-safe/liboqs/jobs/252002293) that reverted the last commit to a previous version of master.

**Comment by dstebila** [comment]
We just discussed this in our weekly call, @smashra will be looking into it.

**Comment by vsoftco** [comment]
Fixed it by replacing .global in all asm files with .globl (that's because macOS uses some ancient "as" tool)

**Comment by dstebila** [comment]
Fixed in #152

**Comment by dstebila** [comment]
Still borken.
